### PR TITLE
Remaking Crowdin and node 15

### LINKED ISSUE #1290: Remake the entire i18n system
it sucks

PS: add RTL support for some languages

### PR DESCRIPTION
**Pull request by jaxxibae**
- Closes #1290 

_(WIP/RFR? @Doges)_

### CODE REVIEW

**Comment by Doges** [comment]
still wip

**Comment by jaxxibae** [comment]
What needs to be done, Doges?

**Comment by Doges** [comment]
https://github.com/SwitchbladeBot/switchblade/issues/1290#issue-822202207

**Comment by jaxxibae** [comment]
I don't see what's the problem with i18next..?
The crowdin dependency has been removed, and I'll be looking for RTL support soon

**Comment by Doges** [comment]
> 
> 
> I don't see what's the problem with i18next..?
> The crowdin dependency has been removed, and I'll be looking for RTL support soon

i was talking about the rtl support, but yeah, the i18next is fine

**Comment by jaxxibae** [comment]
RTL support already exists for i18next, it's Discord that handles RTL text poorly imho
image

**Comment by jaxxibae** [comment]
# RFR.

File: package.json [hunk 1]
```diff
@@ -84,6 +83,7 @@
     "steamladder": "^1.0.4",
     "transform-props": "^1.0.1",
     "turndown": "^6.0.0",
+    "unzip": "^0.1.11",
```
Comment (Doges) [inline package.json:86]: what if we use https://www.npmjs.com/package/adm-zip?

Comment (jaxxibae) [reply package.json:86]: We could use that, it uses 0 dependencies so it's better than unzip

**Comment by jaxxibae** [comment]
@Doges RFR

**Comment by Doges** [comment]
now we need to solve the standard errors, nice

**Review by Doges — APPROVED** [review]
LGTM

**Pull request merged**